<launch>
  <!-- you can load an exisiting tree with <node ... args="tree.bt"> !-->
  <node
    name="matching_octomap_server"
    type="matching_octomap_server"
    pkg="pandora_slam_3d"
    output="screen">
    <param name="resolution" value="0.05" />
    <param name="frame_id" type="string" value="map" />
    <param name="base_frame_id" type="string" value="base_link" />
    <param name="sensor_model/max_range" value="4.0" />
    <remap from="cloud_in" to="/kinect/depth_registered/points/matched" />
  </node>

</launch>
